backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java

```diff
@@ -74,8 +76,16 @@ public void run() {
       // Update repository timestamp
       deleteRepositories(session);
     } catch (Exception ex) {
-      ex.printStackTrace();
-      LOGGER.error("DeleteEntitiesCron Exception: ", ex);
+      if (ex instanceof StatusRuntimeException) {
```

Comment:
we need to refactor the error handling since this way if one project throws exception the entire flow does not proceed.